TestsZipCodeEndpoint.cs

```diff
@@ -0,0 +1,195 @@
+﻿using Newtonsoft.Json;
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using RestSharp;
+
+namespace APIAutomation.Tests
+{
+    [TestFixture]
+    public class ZipCodeEndpointGETTests
+    {
+        private RestClient _client;
+        private RestRequest _request;
+
+        [SetUp]
+        public void Setup()
+        {
+            var client = ClientForReadScope.GetInstance("http://localhost:8000", "0oa157tvtugfFXEhU4x7", "X7eBCXqlFC7x-mjxG5H91IRv_Bqe1oq7ZwXNA8aq");
+            _client = client.GetRestClient();
+            _request = new RestRequest("/zip-codes", Method.Get);
+        }
+
+        [Test]
+        public void GetAllZipCodes_ReturnsAllAvailableZipCodes()
```

Comment:
Don't forget to add _Test postfix for each your test name method

Reply:
Ok, it is done